portfolio/src/main/webapp/script.js

```diff
@@ -22,7 +22,21 @@ $(document).ready(function() {
 
     let $height = $("header").outerHeight(true);
 
-    $("#main-container").css("margin-top", `${$height + 20 + "px"}`);
+    $("#main-container").css("margin-top", `${$height + 30 + "px"}`);
```

Comment:
I'm not super css-fluent, so I'll take this as a chance to learn.  Why do you need to set this in javascript?  Is there something dynamic going on here that you're reacting to?

Reply:
Yes it is dynamic because the height of the header changes sometimes and I also wanted to add an additional offset from the top of the main content since the header is fixed to the top of the page.